src/mmw/js/src/modeling/models.js

```diff
@@ -194,9 +194,9 @@ var ScenarioModel = Backbone.Model.extend({
     },
 
     initialize: function() {
-        // TODO: When fetching model or populating from a preloaded source
-        // this attribute should not be initialized as empty.
-        this.set('modifications', new ModificationsCollection());
+        if (this.get('modifications') === null || "[]") {
```

Comment:
On second thought, I'm going to file an issue. There's currently a problem with the way the modifications are saved.